tests/test_extract_corpora.py

```diff
@@ -0,0 +1,32 @@
+import shutil
+import os
+from pathlib import Path
+from . import helper
+from silnlp.common.paratext import extract_project, extract_term_renderings
+
+
+pp_dir = Path(__file__).parent / "Paratext/projects"
+
+scr_dir = Path(__file__).parent / "MT/scripture"
+scr_truth_dir = Path(__file__).parent / "MT/scripture_truth"
+
+
+def test_extract_corpora():
+    shutil.rmtree(scr_dir, ignore_errors=True)
```

Comment:
I think it would be good to use `tempfile.TemporaryDirectory` here and set `MT_SCRIPTURE_DIR` to the temp dir path. Maybe something like this:

```
with tempfile.TemporaryDirectory() as src_dir:
    MT_SCRIPTURE_DIR = Path(src_dir)
```

I think this would be safer.